metashare/api/sf_run_flow.py

```diff
@@ -46,19 +49,32 @@ def capitalize(s):
     return s[0].upper() + s[1:]
 
 
-def refresh_access_token(*, config, org_name):
+def refresh_access_token(*, config, org_name, scratch_org):
     """
     Construct a new OrgConfig because ScratchOrgConfig tries to use sfdx
     which we don't want now -- this is a total hack which I'll try to
     smooth over with some improvements in CumulusCI
     """
-    org_config = OrgConfig(config, org_name)
-    org_config.refresh_oauth_token = Mock()
-    info = jwt_session(
-        SF_CLIENT_ID, SF_CLIENT_KEY, org_config.username, org_config.instance_url
-    )
-    org_config.config["access_token"] = info["access_token"]
-    return org_config
+    try:
+        org_config = OrgConfig(config, org_name)
+        org_config.refresh_oauth_token = Mock()
+        info = jwt_session(
+            SF_CLIENT_ID, SF_CLIENT_KEY, org_config.username, org_config.instance_url
+        )
+        org_config.config["access_token"] = info["access_token"]
+        return org_config
+    except HTTPError as err:
+        error_msg = "Are you certain that the org still exists?"
+
+        if get_current_job():
+            job_id = get_current_job().id
+            error_msg += f" If you need support, your job ID is {job_id}."
+        else:
+            error_msg += f" {err.args[0]}"
+
+        err = err.__class__(_(error_msg), *err.args[1:],)
```

Comment:
@wlonk Is this approach (wrapping the final `error_msg` for translation, instead of the pieces) acceptable?

Reply:
It works, but:
>(The caveat with using variables or computed values, as in the previous two examples, is that Django’s translation-string-detecting utility, `django-admin makemessages`, won’t be able to find these strings. More on `makemessages` later.)

from https://docs.djangoproject.com/en/2.2/topics/i18n/translation/

But we're not using Django's `makemessages`, so I think this is OK.

Reply:
Aren't we though?

https://github.com/oddbird/MetaShare/blob/master/derrick#L67-L70

Reply:
I thought we were building messages in JS? I guess we are using `makemessages`; I think we should use string literals as much as we can, then.

Reply:
For strings that are defined in JS, we're building them in JS. For strings that come from the API, those need to be built by the API.

Reply:
@wlonk Are you willing to fix that? I noticed some in `consumers.py` too.

Reply:
Yep.

Reply:
I don't see anything in `consumers`; I see some strings that we can't further simplify, and none of them are directly user-facing anyway. Am I missing anything?

Reply:
I was referring to https://github.com/SFDO-Tooling/MetaShare/blob/master/metashare/consumers.py#L68-L70 and https://github.com/SFDO-Tooling/MetaShare/blob/master/metashare/consumers.py#L76-L78, but if those are okay then 👍

Ready to merge once the py test is fixed.

Reply:
Had to revert to your approach, can't concat string-proxies returned by `gettext`.